src/verify.c

```diff
@@ -252,6 +252,7 @@ static int get_required_files(struct manifest *official_manifest, struct list *s
 	if (cmdline_option_install) {
 		progress_set_next_step("download_packs");
 		print("\n");
+		progress_report_interdeterminate();
```

Comment:
Can't we use the spinner here? So cmdline users would benefit from that too? Let's try to add spinners on any operation that takes a long time.